data/json/item_groups.json

```diff
@@ -6560,6 +6560,7 @@
       ["bio_painkiller", 10],
       ["bio_alarm", 10],
       ["bio_ears", 10],
+	   ["bio_earplugs", 10],
```

Comment:
Uh, you actually didn't solve the issue (see below), but it seems that it's better to focus on the general discussion of this PR before you apply any more efforts to solving this (actually minor) 'TAB-to-space' issue.
![untitled](https://cloud.githubusercontent.com/assets/5421954/11610385/acf7c11e-9bb2-11e5-951d-3ab322cd76f5.png)